upup/pkg/fi/cloudup/awsup/machine_types.go

```diff
@@ -837,7 +800,7 @@ var MachineTypes []AWSMachineTypeInfo = []AWSMachineTypeInfo{
 	{
 		Name:              "i3.metal",
 		MemoryGB:          512,
-		Cores:             64,
+		Cores:             72,
```

Comment:
The table [here](https://aws.amazon.com/ec2/instance-types/i3/) states "i3.metal provides 72 logical processors on 36 physical cores". So this is correct.